### DISCUSSION TITLE
RamOS

### DISCUSSION THREAD

**Discussion by YesTheProxy** [theshmunguss1/RamHub#11] [Ideas]
So @theshmunguss1 and I are working on some concept art for an in-browser MacOS Like version of ram networks, here's a design
![macos_243-3](https://github.com/theshmunguss1/RamHub/assets/131057815/cbff8254-a8ed-4834-bf6c-8d455706d249)

**Comment by YesTheProxy** [comment]
Finder will be a link to GitHub, then hammerhead, E-Mail is skiff, astroid is well, astroid (Credit to @VyperGroup) photos, facetime, reminders, activity, adobe, trash, folder, notes, and messages logo will be removed, the calendar is a calendar, contacts is our contact info Music will be Spotify Unblocked, podcast will take you to my twitch/YT for the podcast, News will be newsletters, drawing will open a drawing app (Pixel art, pixlr, etc.) the app store (Might) add more games or proxies, settings will have some settings (Wallpaper, dark mode, etc.)

**Reply by DJshelfmushroom** [reply]
I could def help with that, maybe make a repo for us to commit to?

**Reply by theshmunguss1** [reply]
@YesTheProxy It wouldn't be a _direct_ clone of MacOS, but probably a design language similar to it.
Though I don't really like an OS where every icon leads to a RamWorks thing, so maybe we could have more proxies, a TV/movie app, ect. An idea that would be cool, you could download files & keep Finder as it is rather than link it to Github. From a technical perspective, a browser cache is nowhere near large enough to reliably store files, so you could download a file from a proxy as the file address.
![flowchart](https://github.com/theshmunguss1/RamHub/assets/142841882/ecaf63b4-6c18-4b37-b8dd-e7dff899ef00)

**Reply by YesTheProxy** [reply]
That would be pretty cool!

**Comment by YesTheProxy** [comment]
> I could def help with that, maybe make a repo for us to commit to?

sure!

**Comment by YesTheProxy** [comment]
Ok, I found this simulator https://macos-web.app/ it's open source so we can change what we'd like but we need to give credit

**Reply by DJshelfmushroom** [reply]
hmmm. I'll check it out!

**Reply by DJshelfmushroom** [reply]
it's in svelte though

**Reply by theshmunguss1** [reply]
From [their website](https://svelte.dev/): _"Write breathtakingly concise components using languages you already know — HTML, CSS and JavaScript. Oh, and your application bundles will be tiny as well."_

**Reply by theshmunguss1** [reply]
Their docs are really good, we can use this.

**Reply by YesTheProxy** [reply]
Oh and there's another one https://mhmdmhd6.github.io/Mac-OS-Desktop/ this is coded in HTML, CSS, And Javascript

**Reply by theshmunguss1** [reply]
> this is coded in HTML, CSS, And Javascript
So it's interchangeable with svelte. I like the first one more

**Reply by YesTheProxy** [reply]
same

**Comment by WiFiGiver** [comment]
What happened to this concept in the public one? This is either obviously not MacOS or just very far modded. Did you start from scratch @DJshelfmushroom?

**Reply by DJshelfmushroom** [reply]
yes
